_data/apis.json

```diff
@@ -394,7 +394,8 @@
         "glossary": true,
         "formality": true,
         "adaptive": true,
-        "self-serve": true
+        "self-serve": true,
+        "file_translation": true
```

Comment:
We should probably make an issue to change this from boolean to a list of supported file types.

Reply:
done #613